gizmo.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+GIZMO Challenge
+
+@author: jenni.saaristo@helsinki.fi
+@version: 20210319
+@notes: Funfunfun!
+"""
+
+import numpy as np
+import pandas as pd
+
+
+# -------------------- Exercise 1 --------------------
+
+def hello(name, country='Finland'):
+    print('Hello ' + name + ', how are things in ' + country + '?')
```

Comment:
Alternatively, you can use Python f-strings:

```python
print(f'Hello {name}, how are things in {country}?')
```

Reply:
Awesome, thanks for the tip!